src/web-ui/src/public/Checkout.vue

```diff
@@ -15,157 +10,79 @@
       </div>
 
       <div class="container" v-if="cart">
-        <div class="row justify-content-center" v-if="showCheckout == true">
-          <div class="alert alert-secondary">This storefront is not real. Please do not enter actual billing information. Your order will not be fulfilled.</div>
-        </div>
         <div class="row text-left" v-if="showCheckout == true">
-          <div class="col-md-4 order-md-2 mb-4">
-            <h4 class="d-flex justify-content-between align-items-center mb-3">
-              <span class="text-muted">Summary</span>
-              <span class="badge badge-secondary badge-pill">{{ this.cartQuantity }}</span>
-            </h4>
-            <ul class="list-group mb-3">
-              <li class="list-group-item d-flex justify-content-between">
-                <span>Sub Total (USD)</span>
-                <strong>${{ this.cartSubTotal.toFixed(2) }}</strong>
-              </li>
-              <li class="list-group-item d-flex justify-content-between">
-                <span>Tax (USD)</span>
-                <strong>${{ this.cartTaxRate.toFixed(2) }}</strong>
-              </li>
-              <li class="list-group-item d-flex justify-content-between">
-                <span>Shipping (USD)</span>
-                <strong>${{ this.cartShippingRate.toFixed(2) }}</strong>
-              </li>
-              <li class="list-group-item d-flex justify-content-between">
-                <span>Total (USD)</span>
-                <strong>${{ this.cartTotal.toFixed(2) }}</strong>
-              </li>            
-            </ul>
-            <form class="card p-2">
-              <div class="input-group">
-                <input type="text" class="form-control" v-model="order.promo_code" placeholder="Promo code">
-                <div class="input-group-append">
-                  <button type="submit" class="btn btn-secondary">Redeem</button>
-                </div>
+          <div class="col-md-4 order-md-2">
+            <div class="card p-1">
+              <div class="card-body">
+                <h4 class="d-flex justify-content-between align-items-center mb-3 card-title text-muted">
+                  Order Summary
+                </h4>
+                <ul class="list-group list-group-flush mb-3">
+                  <li class="list-group-item p-1 d-flex justify-content-between">
+                    <span>{{ this.cartQuantity }} item{{ this.cartQuantity === 1 ? '' : 's' }} in cart</span>
+                    <strong>${{ this.cartTotal.toFixed(2) }}</strong>
+                  </li>
+                </ul>
+                <button class="checkout-btn btn btn-outline-dark btn-block btn-lg btn-block" v-on:click="submitOrder">Place your Order</button>
               </div>
-            </form>
+            </div>
+
+            <button v-if="true || pinpointEnabled && user" v-on:click="triggerAbandonedCartEmail" class="abandoned-cart-btn btn btn-primary btn-lg m-4">
+              Trigger Abandoned Cart email
```

Comment:
Since this button is looking pretty funky on medium width screens before the mobile layout kicks in, let's either use auto-sizing similar to the cart page for this column or use the mobile layout sooner.